scraper_service/main.py

```diff
@@ -1,15 +1,20 @@
 import shutil
 from time import time
+from random import randint
+from datetime import datetime, timezone
 
 import requests
 from fake_useragent import UserAgent
 
 
 def scrape_handler():
-    now = int(time())
+   
+    timestamp = datetime.now()
+    timestampString = str(timestamp.date()) + "_" + str(timestamp.hour).zfill(2) + "-" + str(timestamp.minute).zfill(2) \
```

Comment:
You might find this convenient.
`timestamp_str = timestamp.strftime("%Y_%m_%d_%H%M_%S")`

Would it be ok to stick with `variable_with_under_score` and `function_name` format?
More info here https://pep8.org/

Reply:
thanks for the tip.  I will revise later tonight